libr/core/bin.c

```diff
@@ -418,46 +418,30 @@ static int is_executable (RBinObject *obj) {
 #define DBSPATH R2_LIBDIR"/radare2/"R2_VERSION"/fcnsign"
 
 void r_anal_type_init(RCore *core) {
-	const char *anal_arch = r_config_get (core->config, "asm.arch");
+	const char *anal_arch = r_config_get (core->config, "anal.arch");
```

Comment:
there is definitely, try to change it to arm and perform an analysis on an x86 binary :P